### DISCUSSION TITLE
Drag Re-Order Project View Tabs

### DISCUSSION THREAD

**Discussion by stoutgz** [community/community#41782] [Projects and Issues]
### Select Topic Area

Bug

### Body

[Dragging View Tabs to re-order them in a Project](https://github.blog/changelog/2021-11-18-the-new-github-issues-11-18-update/) is broken. I don't seem to be able to find a drag cursor and I have tried clicking and dragging in just about every pixel I could.

- OS: macOS 13.0.1
- Browsers:
    - Brave Browser [Version 1.46.134 Chromium: 108.0.5359.94 (Official Build) (arm64)](https://brave.com/latest/)
    - Safari Version 16.1 (18614.2.9.1.12)
    

https://user-images.githubusercontent.com/3385153/207474084-959a1764-02fc-4fcd-b6e2-7d0b0b739864.mov

**Comment by ryandeussing** [comment]
I duped this here: https://github.com/community/community/discussions/42685

But it actually works, it's just that the UI doesn't really show a standard affordance.

**Reply by stoutgz** [reply]
Thanks for the follow-up. Yes, I seem to be able to get the drag to work under your specified conditions. Good as a work-around, but needs improvement.
